tests/smoke/sandbox/task.yaml

```diff
@@ -62,7 +62,14 @@ execute: |
             print(errno.errorcode[get_errno()])
     EOF
     echo "Running random syscall gives ENOSYS normally"
-    python3 sec.py 22082007 | MATCH ENOSYS
+    if [ "$(uname -m)" = "armv7l" ]; then
+        # armhf kills the process with si_signo=SIGKILL, si_code=ILL_ILLTRP
+        python3 sec.py 22082007 || ret="$?"
```

Comment:
Looks like ENOSYS or SIGILL depends on the syscall number you roll: https://elixir.bootlin.com/linux/v5.0-rc4/source/arch/arm/kernel/traps.c#L596